modules/tensor_mechanics/src/materials/RecomputeRadialReturn.C

```diff
@@ -140,6 +140,18 @@ RecomputeRadialReturn::getIsotropicShearModulus()
   return shear_modulus;
 }
 
+Real
+RecomputeRadialReturn::getIsotropicBulkModulus()
+{
+  Real shear_modulus = getIsotropicShearModulus();
+  Real dil_modulus = _elasticity_tensor[_qp](0,0,0,0);
```

Comment:
It's dilatational modulus, defined as lambda plus two times the shear modulus.  It is simply the entry at (0,0,0,0) in the elasticity tensor.  In another sense, dil_modulus is just a temporary variable to help us get the bulk modulus.  Would a comment be better than changing the variable name?  Both?